PlaybackLab/TVOSSwiftSampleApp/TVOSSwiftSampleApp/List/VideoTableViewController.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  ViewController.swift
+//  TVOSSwiftSampleApp
+//
+//  Copyright © 2018 Ooyala. All rights reserved.
+//
+
+import UIKit
+
+class VideoTableViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+  
+  var tableOptions : Array<PlayerSelectionOption>!
+  var tableView: UITableView!
+  
+  var rowHeight: CGFloat!
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    tableView = UITableView(frame: CGRect(x: 0, y: 0, width: self.view.frame.width, height: self.view.frame.height), style: .plain)
+    self.view.addSubview(tableView)
+    
+    self.tableView.dataSource = self
+    self.tableView.delegate = self
+    
+    self.rowHeight = 100
+    self.populateOptions()
```

Comment:
In Swift usually there's no need in using explicit `self`

Reply:
Done